lib/android/audio.nit

```diff
@@ -364,53 +368,79 @@ class SoundMP
 		self.media_player = media_player
 	end
 
-	redef fun play do self.media_player.start
+	redef fun play do media_player.start
+	redef fun pause do media_player.pause
+	redef fun resume do play
 end
 
 redef class App
-
+	
+	# Sounds handled by the application, when you load a sound, it's added to this list
+	# The list used in the pause and resume functions of the app
+	private var sounds = new Array[Sound] 
+
+	# Returns the default MediaPlayer of the application
+	# When you load a music, it goes in this MediaPlayer
+	# use it for advanced sound management
```

Comment:
Ends the previous sentence with a dot `.` and put a capital at the start of this one.